filefinder/_filefinder.py

```diff
@@ -651,6 +651,37 @@ def search(self, **query):
         df = self._get_subset(**query)
         return type(self)(df)
 
+    def concat(self, other):
+        """concatenate two FileContainers
+
+        Parameters
+        ----------
+        other : FileContainer
+            The other FileContainer to concatenate.
+
+        Returns
+        -------
+        FileContainer
+            The concatenated FileContainer.
+
+        Raises
+        ------
+        ValueError
+            If the other object is not a FileContainer.
+        ValueError
+            If the two FileContainers do not have the same keys.
+        """
+
+        if not isinstance(other, FileContainer):
+            raise ValueError("Can only concatenate two FileContainers.")
+
+        if self.df.columns is not other.df.columns:
+            raise ValueError("FileContainers must have the same keys.")
+
+        ret = copy.copy(self)
+        ret.df = pd.concat([self.df, other.df])
+        return ret
```

Comment:
```suggestion
        df = pd.concat([self.df, other.df])
        return type(self)(df)
```

`type(self)` is `FileContainer`